solitude/base.py

```diff
@@ -719,4 +738,14 @@ class NonDeleteModelViewSet(
     Same as the ModelViewSet, without the DeleteMixin. Uses our local mixins
     to give us ETag support.
     """
-    pass
+
+    # Allow tastypie resources to connect to viewsets, to delete
+    # once tastypie is gone. You really don't want to know.
+    def get_via_uri(self, uri, request=None):
```

Comment:
This (and the methods below) is... weird. I guess you don't have much choice, but if it helps, here is one of the pieces we used to have during the tastypie->DRF transition in zamboni: https://github.com/mozilla/zamboni/pull/1510/files#diff-4b1e07d8ad7eedebce91784875fef749L432